src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -63,14 +64,58 @@ bool P104_data_struct::begin() {
 # define P104_ZONE_DISP  ';'
 # define P104_FIELD_DISP ','
 
+# define P104_CONFIG_VERSION_V2  0xF000 // Marker in first uint16_t to to indicate second version config settings, anything else if first
+                                        // version.
+                                        // Any third version or later could use 0xE000, etc. The 'version' is stored in the first uint16_t
+                                        // stored in the custom settings
+
+/*
+   Settings layout:
+   Version 1:
+   - uint16_t : size of the next blob holding all settings
+   - char[x]  : Blob with settings, with csv-like strings, using P104_FIELD_SEP and P104_ZONE_SEP separators
```

Comment:
Why implement settings versions for something that hasn't been merged yet?

Reply:
To help existing testers to easily migrate their settings.